mozilla/fx-private-relay
# MPP-2479 - Remove auto scroll when creating a custom mask

This PR fixes #MPP-2479

How to test:

On a mobile device and a premium account with a subdomain set, check that creating a custom mask does not scroll up.

https://user-images.githubusercontent.com/13066134/228642978-2e5866b9-239d-4c9f-ad00-a555848bc425.mov
